spec/61-span-id-format.md

```diff
@@ -0,0 +1,20 @@
+## Considerations for `span-id` field generation
+
+This section suggests some practices to consider when implementing
+`span-id` generation algorithms to ensure
+interoperability between different systems.
+
+### Uniqueness of `span-id`
+
+The value of `span-id` MUST be unique within a <a>distributed trace</a>.
```

Comment:
```suggestion
The value of `span-id` SHOULD be unique within a <a>distributed trace</a>.
```
there is no way to guarantee it using random generators

Reply:
even though it is informative section I tend to agree that MUST here is too strong of a requirement